git-proxy/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
-storageService: # s3/gcs/blob/objectstorage
+storageService: "" # s3/gcs/blob/objectstorage
```

Comment:
revert

Reply:
This is needed for the template to pass validation:
[ERROR] templates/: template: git-proxy/templates/secret.yaml:18:7: executing "git-proxy/templates/secret.yaml" at <eq .Values.storageService "objectstorage">: error calling eq: incompatible types for comparison